packages/manager/src/features/Databases/utilities.ts

```diff
@@ -213,6 +216,35 @@ export const toDatabaseFork = (
   return fork;
 };
 
-export const getDatabasesDescription = (database: DatabaseInstance) => {
+export const databaseEngineMap: Record<Engine, string> = {
+  mongodb: 'MongoDB',
+  mysql: 'MySQL',
+  postgresql: 'PostgreSQL',
+  redis: 'Redis',
+};
```

Comment:
To prevent these from being mutated
```suggestion
} as const;
```

Reply:
Done